From 30112aeea85f1cc1c9cc14d303aca0265e91c610 Mon Sep 17 00:00:00 2001
From: LongNguyen <long.nguyenvancmc@toshiba-tsdv.com>
Date: Wed, 20 Dec 2017 10:36:26 +0700
Subject: [PATCH 2/3] edit longnguyen

---
 rebase.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rebase.sh b/rebase.sh
index 023b238..040e598 100644
--- a/rebase.sh
+++ b/rebase.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 #edit longnguyen1
-
 echo "test rebase fucking stupid"
+#edit longnguyen
 
 exit
-- 
2.1.4

